enterprise/backend/src/metabase_enterprise/semantic_search/index.clj

```diff
@@ -5,6 +5,7 @@
    [honey.sql.helpers :as sql.helpers]
    [metabase-enterprise.semantic-search.embedding :as embedding]
    [metabase.models.interface :as mi]
+   [metabase.search.appdb.specialization.postgres :as postgres-spec]
```

Comment:
kondo complains about importing from this module, not sure what is the best remedy here